src/main/java/uk/gov/pay/directdebit/notifications/services/UserNotificationService.java

```diff
@@ -44,10 +45,12 @@ public void sendMandateFailedEmailFor(Mandate mandate) {
 
     public void sendOnDemandMandateCreatedEmailFor(Mandate mandate) {
         adminUsersClient.sendEmail(EmailTemplate.ON_DEMAND_MANDATE_CREATED, mandate,
-                ImmutableMap.of(
-                        MANDATE_REFERENCE_KEY, mandate.getMandateReference(),
-                        DD_GUARANTEE_KEY, directDebitConfig.getLinks().getDirectDebitGuaranteeUrl()
-                )
+                ImmutableMap.<String, String>builder()
+                        .put(MANDATE_REFERENCE_KEY, mandate.getMandateReference())
+                        .put(BANK_ACCOUNT_LAST_DIGITS_KEY, BANK_ACCOUNT_MASK_PREFIX + mandate.getPayer().getAccountNumberLastTwoDigits())
```

Comment:
why do we have to send a load of asterisks. can't that just be in the template? Not a big deal

Reply:
There was a bug in our email provider before:
If we define more than two asterisks in the template it was truncated to two, that is why we are passing the asterisks from our code.
Katie will verify if the bug still exists and we can address the changes in the next PR.